examples/cube_asyncio.py

```diff
@@ -0,0 +1,31 @@
+"""
+Cube asyncio
+------------
+
+Run a wgpu example on the glfw backend, and the asyncio loop
+"""
+
+import asyncio
+
+from rendercanvas.glfw import RenderCanvas
+from rendercanvas.asyncio import loop
+from rendercanvas.utils.cube import setup_drawing_sync
+
+
+# The asyncio loop is the default, but this may change, so better be explicit.
+RenderCanvas.select_loop(loop)
```

Comment:
Setting the loop instance feels more explicit, and you need it in scope anyway, so you can `run()` it. Plus it allows passing custom loops, e.g. in tests.

Reply:
Basically I want to avoid:
```py
from rendercancas.glfw import RenderCanvas, loop

RenderCanvas.select_loop("trio")

loop.run() 
```
Nothing happens, because the canvas is not on the asyncio-loop.

```

Reply:
And is it possible to write `asyncio.run(main())` to start their app for asyncio users?

Reply:
Yes, see the new `cube_asynci.py`  and `cube_trio.py`  examples.